### DISCUSSION TITLE
1.21.4 Updated Datapack Version 61

### DISCUSSION THREAD

**Discussion by MeWithNoGoodUsername** [SPGoding/datapack-language-server#1699] [Q&A] [answered]
Just wondering when we will be seeing an updated version that includes the changes made to the datapack format for the 1.21.4 update.

[Minecraft Java Edition 1.21.4 _ Datapack V61 Minecraft.pdf](https://github.com/user-attachments/files/18283708/Minecraft.Java.Edition.1.21.4._.Datapack.V61.Minecraft.pdf)

**Comment by misode** [comment, marked as answer]
The extension is fully updated to 1.21.4. Just make sure that you have set your `pack_format` to 61 and then reloaded (closed and reopened vscode)

**Reply by MeWithNoGoodUsername** [reply]
![image](https://github.com/user-attachments/assets/1635aa37-d268-4076-b4da-c3691e3230d0)
![image](https://github.com/user-attachments/assets/8e81c6bc-89c9-48fb-a54d-327cfc1d25e1)
![image](https://github.com/user-attachments/assets/742c2e1d-2680-449b-b995-699ef7dd6b5e)

pack.mcmeta: "pack_formet" is 61
spyglass.json "gameVersion is 1.21.4
Spy Glass is up to date. 

Something is wrong with my version.
